Platon/backend/app/search_engine/helpers.py

```diff
@@ -99,7 +99,11 @@ def remove_punctuation(search_query):
 
     @staticmethod
     def sort_ids(related_id_score_list):
-        return sorted(related_id_score_list, key=lambda tup: tup[1])
+        return sorted(related_id_score_list, key=lambda tup: tup[1],reverse=True)
+
+    @staticmethod
+    def sort_results(results_list, key_list, reverse):
+        return sorted(results_list, key=lambda x: " ".join([x[key].lower() for key in key_list]), reverse=reverse)
```

Comment:
These 2 are the functions that I implemented. Why did you change them? If you changed them user search will not be able to sort the results according to their relevance or alphabetical order.